rfc/rfc004-package-log.md

```diff
@@ -12,8 +12,21 @@ http://peter.bourgon.org/go-kit/#package-log
 
 ## Scope
 
-To be defined.
+* Log SHALL be drop in compatible with the standard library logger.
```

Comment:
@aybabtme you're referring to [this](https://github.com/Sirupsen/logrus/blob/38a6ff9215d472ce953bb9aa37b5327715c0f940/writer.go#L9-16)? I guess it ends up logging with an empty fieldset?